content/docs/learning-guide/tutorials/entities-and-prefabs/spawn-a-prefab.md

```diff
@@ -0,0 +1,151 @@
+
+    {{< video src="/images/learning-guide/tutorials/entities-and-prefabs/add-on-graph-start.mp4" info="Detach a prefab instance." autoplay="true" loop="true" >}}
+
+1. Add a **Create Spawn Ticket** node to the graph canvas. Use the search field at the top of the Node Palette to filter the node list. **Drag** a Create Spawn Ticket node into the graph canvas to the left of the On Graph Start node.
+
+    {{< note >}}You can also **right-click** in the graph canvas to access the node list.
+    {{< /note >}}
+
+1. Connect the nodes. **Drag** from the **Out** pin on the On Graph Start node to the **Create Ticket** pin on the Create Spawn Ticket node to connect the nodes.
+
+    {{< video src="/images/learning-guide/tutorials/entities-and-prefabs/connect-nodes.mp4" info="Click and drag to connect Script Canvas nodes." autoplay="true" loop="true" >}}
+
+1. Add the prefab reference to the Create Spawn Ticket node. On the Create Spawn Ticket node, **right-click** on the **Prefab** property and select **Convert to Reference** from the context menu. The property is automatically set to the **PrefabRef** variable you created in step **2**. You can choose the {{< icon settings.svg>}} settings icon in the **Prefab** field to select a variable reference if necessary.
+
+    ![Set the prefab reference on a Create Spawn Ticket Script Canvas Node.](/images/learning-guide/tutorials/entities-and-prefabs/set-prefab-reference.png)
+
+1. Add a **Spawn** node from the **Prefab/Spawning** node group to the graph canvas and connect it to the Create Spawn Ticket node.
+
+    1. Connect the **Ticket Created** pin from the Create Spawn Ticket node to the **Request Spawn** pin of the Spawn node.
+    
+    1. Connect the **SpawnTicket** pin from the Create Spawn Ticket node to the **SpawnTicket** pin of the Spawn node.
+
+    {{< image-width "/images/learning-guide/tutorials/entities-and-prefabs/connect-spawn-node.png" "900" "Connect the Spawn node to the Create Pawn Ticket node.">}}
+
+1. In addition to spawning prefab instances, the Spawn node also needs to populate the SpawnTicket variable with data. On the Spawn node, **right-click** on the **SpawnTicketOut** pin and select **Convert to Reference** from the context menu. The property is automatically set to the **SpawnTicket** variable you created in step **3**. You can choose the {{< icon settings.svg>}} settings icon in the **SpawnTicketOut** field to select a variable reference if necessary.
+
+    ![Set the prefab reference on a Create Spawn Ticket Script Canvas Node.](/images/learning-guide/tutorials/entities-and-prefabs/spawn-ticket-out-ref.png)
+
+1. The **SpawnTicket** variable must be reset when the spawner entity is deactivated. Add an **On Entity Deactivated** node to clear the **SpawnTicket** variable with the following steps:
```

Comment:
@AMZN-mnaumov , is this step still needed after your fix to stabilization branch?

Reply:
removed